tokens-filter/src/jupiter.rs

```diff
@@ -0,0 +1,134 @@
+use std::{
+    collections::HashMap,
+    sync::{Arc, OnceLock},
+};
+
+use async_trait::async_trait;
+use cached::{Cached, TimedCache};
+use serde::Deserialize;
+use solana_sdk::pubkey::Pubkey;
+use tokio::sync::Mutex;
+
+use crate::CheckToken;
+
+pub static DEFAULT_URL: &str = "https://cache.jup.ag/indexed-route-maps-v3";
```

Comment:
I would suggest to use https://station.jup.ag/docs/apis/token-list-api instead

Reply:
I've did it according this [article](https://p2pvalidator.atlassian.net/wiki/spaces/Wallet/pages/2751168513/Scam+Token+Filtering+v2)

also it has differnt numbers of token: 
https://token.jup.ag/strict - 604 addresses
https://token.jup.ag/all - 835 
and here https://cache.jup.ag/indexed-route-maps-v3 is - 874

If u insist to pick ur variant, can u ask Dmitriy to approve one of this endpoints?

Reply:
It is suggestion, because https://token.jup.ag/all was created for token filtration in mind.

Reply:
We shouldn't mark tokens as `suspicious` if user can swap them via jupiter. As far i understand It means that we should check routes. 

Anyway I don't have enough expertise in this question, thats way I ask you to redirect this suggestion to Dmitriy Korpachev.